### DISCUSSION TITLE
Tablet and/or external GPS recommendations

### DISCUSSION THREAD

**Discussion by MartinHore** [OpenOrienteering/mapper#1999] [Questions & Answers]
I'm quite experienced at mapping, but new to using technology in the field. Currently I'm just using my iPhone with my OOM base map loaded as a KMZ file via the Maprun6 app. This displays my GPS position on my map the screen, and allows me to download my track when I get home, but it's only as accurate as my iPhone's GPS receiver. And of course I can't draw anything on the map in the field. 

Can anyone advise me which Android tablet you find works best for drawing on your OOM map in the field. Do you find you also need an external GPS device to get best accuracy, or can you rely on the tablet's in-built GPS?

Many thanks for any help. And sorry if the question has been answered before. I couldn't find a similar topic.

Martin

**Comment by pkturner** [comment]
Three years ago I got a Samsung  Galaxy Tab Active2 8" Ruggedized Tablet, which works well, even in light rain. I like the S-Pen for drawing on the screen. On the steep hillsides in the forest, its built-in GPS was too-frequently wrong by 5 to 10 m. I augmented it with an inexpensive little Bluetooth GPS device which was better but still not good.

You could also ask the question and look for information on the Facebook group, "Orienteering Mappers Int."

**Reply by MartinHore** [reply]
Thank you Scott. Very helpful. I've just signed on to the Facebook group too.
Martin

**Reply by hedger48** [reply]
> Three years ago I got a Samsung Galaxy Tab Active2 8" Ruggedized Tablet, which works well, even in light rain. I like the S-Pen for drawing on the screen. On the steep hillsides in the forest, its built-in GPS was too-frequently wrong by 5 to 10 m. I augmented it with an inexpensive little Bluetooth GPS device which was better but still not good.
> 
> You could also ask the question and look for information on the Facebook group, "Orienteering Mappers Int."

Thanks for this.  Can you tell me if you have any issues writing with spen when you zoom in?   I zoom in to take note and my writing is very blurry.  Is it a matter of scale?

**Reply by pkturner** [reply]
@hedger48 My s-pen works well when zoomed in. However, I don't "take notes" in Mapper. Primarily I use the "Draw Paths" and the edit tools.

**Reply by mlerjen** [reply]
@hedger48 the blur at the zoomed in sketching is due to the resolution of the scribble layer. If  you want to sketch at a higher zoom level you need a scribble layer with a higher resolution. I found 256dpi suiting best for me.

**Comment by mlerjen** [comment]
The same  here. Samsung Galaxy Note 8. Built-in GPS is often good. External Navilock BT 821 G mounted on my hat is slightly better and battery saving but sometimes still not accurate enough. The alternatives though are much more expensive and not as lightweight. As I never use GPS as sole indication to position stuff which positionaccuracy is crucial, I am fine with it

**Reply by MartinHore** [reply]
Thank you Mierjen. Very helpful.

Martin

**Comment by jjojala** [comment]
Good subject and would require far more attention IMHO 👍 

Ruggered tablet would be of course great, but I've used ordinary, cheap Android tablet (Huawei MediaPad M5 8) with waterproof tablet pouch. Even zip-bag may do the trick. Stylus works decently through the pouch assuming that the pouch is closed so that there is as little air as possible in the pouch. Finding a good stylus needed some experimenting. I've tried couple universal active styluses didn't worked as I expected, but the case might be different for brand -specific ones, like S-Pen for Samsung. Currently I'am experimenting Meko disc styluses. Looks funny, but works pretty well.

As for "GPS", I've used GNSS 2000 Plus (also in hat ;-), which is cheap BT-receiver supporting GPS, Glonass, and Galileo therefore suitable at least in Europe. If you happens to have NTRIP correction data available on your area, you may improve the accuracy. I guess there's at least commercial NTRIP providers pretty much everywhere(?) I live in Finland, where we have national land survey agency providing NTRIP services for free ([link](https://www.maanmittauslaitos.fi/en/maps-and-spatial-data/positioning-services/dgnss-service)). With NTRIP and suitable Android app such as [Bluetooth GNSS](https://play.google.com/store/apps/details?id=com.clearevo.bluetooth_gnss) one may turn ordinary GNSS receiver into a bit more accurate [differential gnss](https://gssc.esa.int/navipedia/index.php/DGNSS_Systems). The downside is, that you also need a mobile broadband on the field to get the correction data.

**Comment by hedger48** [comment]
Thank you, I can't find how to change the resolution of a specific layer.

My screen resolution in the setup showed 225 but changing that to 256
didn't do anything that I could tell.

Thx

**Reply by mlerjen** [reply]
On the desktop version: 
> save map as image (define the extent and resolution of the future scribble layer)
> Load the image as template
> empty the image with the eraser&area tool of the scribble palette.

**Comment by sfroyen** [comment]
The TFT display on the Active tablets (I'm using an Active3) is much better in sunlight than the OLED display on the Tab S tablet that I used previously. The S-pen is also much more precise. The new pen-only setting for the OOM Android app is an added benefit. I added a $20 protective case because I found the provided case too slippery. I also got the modem version in order to receive RTK data.

For GNSS receiver I use a Ublox ZED-F9P. It supports dual band GPS (L1, L2), Galileo (E1, E5), Beidou (B1, B2) and Glonas (L1) plus SBAS and RTK. The accuracy is usually  better than 1m using SBAS and 20cm using RTK. I have two versions of this receiver. The first is a C099-F9P eval module which contains everything you need (except a battery and a case). This is the least expensive option. The other is a SparkFun GPS-RTK2 board (there are other similar boards). This solution is more expensive and requires additional components and soldering, but can be assembled into a must smaller package.

The GNSS receiver in the tablet is also quite accurate. I've seen some information the suggests it should be possible to combine raw GNSS data from an Android device with external RTK data. Perhaps this could be built into OOM?

**Comment by hedger48** [comment]
Thank you, this worked great!
 I finally got out to the field using Galaxy tab s6 lite with s pen, the
app was really great to work with!  Loved using it for field checking. Love that it follows my location and I can scribble notes right on a map template. 

Also in the field  I carried Garmin gpsmap64 and compared the tracks from the Samsung tablet, the tablet performed very well.  

tablet has   A-GPS, GLONASS, BDS, GALILEO, no cellular radio on it.

![AFA28C00-4FAB-4E36-8E9E-7AB46C4D5BA8](https://user-images.githubusercontent.com/47093614/148651604-9666c6d6-109b-4944-bfbd-72dc9f928479.jpeg)



On Sun, Dec 12, 2021 at 2:12 AM mlerjen ***@***.***> wrote:

> On the desktop version:
>
> save map as image (define the extent and resolution of the future scribble
> layer)
> Load the image as template
> empty the image with the eraser&area tool of the scribble palette.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenOrienteering/mapper/discussions/1999#discussioncomment-1792748>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALHJO3S7D5YOM6F3YBPHZO3UQRKNLANCNFSM5HLFTQXQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
Kelly

**Comment by jjojala** [comment]
Old subject, but always topical as devices are constantly evolving...

Have to promote Samsung Active Tab's. I have two of them, old Tab Active2 (8") is light to carry and the screen is usually large enough. The downside is that it is already very old, you'll need to use OOM binaries especially build for older devices (though they're generally available). Also the battery lasts roughly four hours... (though, that is usually enough for me).

Another one is Tab Active4 Pro (10"), which is slightly heavy to carry, but the screen is definetely large enough and the battery lasts maybe twice as long as in Tab2 Active2 (and definetely long enough).

Both have Samsung S-Pen, which I have no bad to say about. Very precise. Mapping is possible practically in all conditions, except in heavy rain, where I have had to end mapping few times (as the drain drops start to "make mapping" too...).

For "budjet" GNSS one option is [ArduSimple's RTK Portable Bluetooth Kit](https://www.ardusimple.com/product/simplertk2blite-bt-case-kit/). With RTK correction data, one can achieve accuracy available only in pro-level devices. The kit is not ready for field as such, but one must build some way to carry and protect it from weather.

**Comment by LadaHo** [comment]
In my experience, a good choice is the RTK GNSS UM980 module (more accurate and cheaper than the U-Blox F9P) with a fullband helix antenna, supplemented by a Bluetooth module HC-05 and a Li-Ion battery. All in a waterproof box printed on a 3D printer. For inspiration, you can follow https://github.com/lpechacek/u-blox-gnss-receiver.
